offchain/danaswap-ui/plugins/compile-ejs-to-html.js

```diff
@@ -7,13 +7,11 @@ const getPathByDir = (dir) => {
   return routes.find((route) => dir.endsWith(route.dir))?.path
 }
 
-module.exports = (options = {}) => ({
+module.exports = ({ dest, data = {}, options = {}) => ({
```

Comment:
Cannot find the matching curly brace

Reply:
yep... I missed the and ending one... I lost the commit and thought I could hack it together real quick like